kotlintest-extensions/src/main/kotlin/io/kotlintest/extensions/system/MapOverrides.kt

```diff
@@ -1,12 +1,34 @@
 package io.kotlintest.extensions.system
 
+
+enum class SystemOverrideMode { ALLOW_OVERRIDE, DENY_OVERRIDE }
```

Comment:
The two possibilities, actually lets say three.

If already set ignore
If already set override
Error if already set

So maybe the nomenclature can be:

- SET_OR_OVERRIDE
- SET_OR_ERROR (default)
- SET_OR_IGNORE

?

Reply:
My only point to this is that `SET_OR_OVERRIDE` should be set as default. It's the default now, and I don't think we need to change that behavior and potentially break clients

Reply:
Yes ok agreed.